src/frontend/apps/standalone_site/src/features/HomePage/components/HomePage.tsx

```diff
@@ -44,21 +44,21 @@ const TextBannerBox = styled(Box)`
 `;
 
 interface LayoutProps {
-  isFullLayout: boolean;
+  $isFullLayout: boolean;
```

Comment:
FMI, why adding a `$` here ?

Reply:
All the props are propagating to the component `Heading` then the DOM, but we don't want `isFullLayout` to propagate because it is used only with the` styled` component `HeadingBanner`, so to block the propagation you can add a dollar in front of your variable (https://styled-components.com/docs/basics#passed-props).

Reply:
Oh OK, thanks !